src/FlapperUniV2SwapOnly.sol

```diff
@@ -97,8 +96,7 @@ contract FlapperUniV2SwapOnly {
         wards[msg.sender] = 1;
         emit Rely(msg.sender);
 
-        // Initial values for safety
-        hop  = 1 hours;
+        // Initial value for safety
```

Comment:
neat - can put the comment in the same line now.

Reply:
Changed in 79a0021c